src/components/theme/Banner/View.jsx

```diff
@@ -0,0 +1,141 @@
+    [content, moment],
+  );
+  const image = getImageSource(content['image']);
+
+  return (
+    <Banner {...props}>
+      <div
+        className="image"
+        style={image ? { backgroundImage: `url(${image})` } : {}}
+      ></div>
+      <div className="gradient">
+        <Container>
+          <Banner.Content
+            actions={
+              <>
+                {!hideShareButton && (
+                  <Popup
+                    className="share-popup"
+                    content={() => (
+                      <>
+                        <p>Share to:</p>
+                        <div className="actions">
+                          <Banner.Action
+                            icon="ri-facebook-fill"
+                            color="blue"
+                            onClick={() => {
+                              sharePage(content['@id'], 'facebook');
+                            }}
+                          />
+                          <Banner.Action
+                            icon="ri-twitter-fill"
+                            color="blue"
+                            onClick={() => {
+                              sharePage(content['@id'], 'twitter');
+                            }}
+                          />
+                          <Banner.Action
+                            icon="ri-linkedin-fill"
+                            color="blue"
+                            onClick={() => {
+                              sharePage(content['@id'], 'linkedin');
+                            }}
+                          />
+                        </div>
+                      </>
+                    )}
+                    position="top center"
+                    size="small"
+                    trigger={
+                      <Banner.Action
+                        icon="ri-share-fill"
+                        title="Share"
+                        className="share"
+                        onClick={() => {}}
+                      />
+                    }
+                  />
+                )}
+                {!hideDownloadButton && (
+                  <Banner.Action
+                    icon="ri-arrow-down-s-line"
+                    title="Download"
+                    className="download"
+                    onClick={() => {
+                      window.print();
+                    }}
+                  />
+                )}
+              </>
+            }
+          >
+            <Banner.Title>{content['title']}</Banner.Title>
+            <Banner.Metadata>
+              <Banner.MetadataField
+                hidden={hideContentType}
+                value={contentType || content['@type']}
+              />
+              <Banner.MetadataField
+                hidden={hideCreationDate}
+                type="date"
+                label="Created"
+                value={creationDate}
+                title="Created on {}"
```

Comment:
We should use i18n messages. At least for public facing pages.

Reply:
https://github.com/eea/volto-eea-website-theme/pull/16/commits/b38a6e8eb0abf6bd5fea3c6c64234c20a3cf4b8e